File: PRAKTIKUM_2/4/PRAK204-2410817110010-DanielNoprianto.py
#case pertama
jari_jari=int(input(""))
tinggi=int(input(""))

selimut = 2*22/7*jari_jari*tinggi
alas = 2*22/7*jari_jari*jari_jari
volume = 22/7*jari_jari*jari_jari*tinggi
luas = selimut+alas
keliling = 2*22/7*jari_jari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}") 

#case kedua
r,t=map(int,input().split())

s = 2*22/7*r*t
a = 2*22/7*r*r
v = 22/7*r*r*t
l = s+a
k = 2*22/7*r

print(f"Volume = {v:.2f}")
print(f"Luas = {l:.2f}")
print(f"Keliling = {k:.2f}") 

File: PRAKTIKUM_2/4/PRAK204-2410817110010-DanielNoprianto.c
#include <stdio.h>
//case pertama
int main(){
    float jari_jari, tinggi, volume, selimut, alas, luas, keliling;

    scanf("%f",&jari_jari);
    scanf("%f",&tinggi);
    
    selimut = 2*22*jari_jari*tinggi/7;
    alas = 2*22*jari_jari*jari_jari/7;
    volume = 22*jari_jari*jari_jari*tinggi/7;
    luas = selimut+alas;
    keliling = 2*22*jari_jari/7;

    printf("Volume= %.2f\n",volume);
    printf("Luas= %.2f\n",luas);
    printf("Keliling= %.2f\n",keliling);

//case kedua
    float r, t, v, s, a, l, k;

    scanf("%f %f",&r,&t);

    s = 2*22*r*t/7;
    a = 2*22*r*r/7;
    v = 22*r*r*t/7;
    l = s+a;
    k = 2*22*r/7;

    printf("Volume= %.2f\n",v);
    printf("Luas= %.2f\n",l);
    printf("Keliling= %.2f\n",k);

    return 0;
}